doc/pylint.rst

```diff
@@ -1,35 +1,95 @@
-The analyzer works with both ``.py`` (or ``.pyw``) Python scripts and whole Python packages (directories containing an :file:`__init__.py` file).
-Then, to run it, select :menuselection:`Sour&ce --> Run static code analysis` from the menubar, click the :guilabel:`Analyze` button in the pane, or press the configurable static analysis shortcut (:kbd:`F8` by default).
-Cancel analyzing a file with the :guilabel:`Stop` button, and if analysis fails, click the :guilabel:`Output` button to find out why.
+=======================
+Using the code analyzer
+=======================
 
-To go directly to the file and line in the :doc:`editor` highlighted by a failed check, just click its name.
-You can click the dropdown or press the :kbd:`Down Arrow` key in the filename field to view results of previous analyses; the number of recent runs Spyder should remember can be customized in the :guilabel:`History` dialog from the :guilabel:`Static Code Analysis` context menu.
+You can select the desired file to analyze directly in the :doc:`editor` by clicking anywhere within it.
+To run the analysis, click the :guilabel:`Analyze` button in the Code Analysis pane, press the configurable shortcut (:kbd:`F8` by default) or select :menuselection:`Source --> Run code analysis` from the menu bar.
 All standard checks are run by default.
-You can turn certain messages off at the line, block or file/module level by adding a ``# pylint: disable=<list of message names>`` comment at the respective level, where the ``<...>`` portion should be replaced with a comma-separated list (or single value) without the ``<``/``>`` of Pylint message names (*e.g.* ``multiple-statements``, or ``fixme, line-too-long``; you can also use error codes like ``C0321`` although names are preferred for clarity).
-Or, you can globally suppress certain messages by editing the :file:`.pylintrc` configuration file in your user home directory.
+To go directly to a line in the :doc:`editor` highlighted by a failed check, just click its name.
+
+.. image:: images/pylint/code_analysis_editor.gif
+   :alt: Spyder Pylint pane, showing running analysis and clicking failed check
+
+|
+
+You can manually enter the Python module or package path you'd like it to check in the path entry box in the pane's toolbar.
+The analyzer works with both individual scripts and whole Python packages (directories containing an :file:`__init__.py` file).
+
+.. image:: images/pylint/code_analysis_file.gif
+   :alt: Spyder Pylint pane, showing running analysis browsing file
+
+|
+
+Cancel analyzing a file with the :guilabel:`Stop` button, and if analysis fails, click the :guilabel:`Output` button to find out why. 
+
+.. image:: images/pylint/code_analysis_stop.gif
+   :alt: Spyder Pylint pane, showing stopping analysis
+
+|
+
+You can click the dropdown or press the :kbd:`Down Arrow` key in the filename field to view results of previous analyses.
+
+
+
+===================
+Contex menu options
```

Comment:
```suggestion
Options menu
```

@juanis2112, in graphical apps context menus are the ones you get when you do a mouse left-click.

Reply:
I discuss this below, this was what I suggested. However, you also need to fix the length of the `=======`s.